pyvista/plotting/camera.py

```diff
@@ -14,7 +14,7 @@ class Camera(_vtk.vtkCamera):
 
     Parameters
     ----------
-    renderer : pyvista.Renderer, optional
+    renderer : pyvista.Renderer, default: None
```

Comment:
This should stay optional since the default is `None`.  What happens when it is `None`?  The current docstring isn't very helpful here.

Reply:
```suggestion
    renderer : pyvista.Renderer, optional
```